VEHICLE/DisableVehicleWeapon.md

```diff
@@ -8,13 +8,13 @@ ns: VEHICLE
 void DISABLE_VEHICLE_WEAPON(BOOL disabled, Hash weaponHash, Vehicle vehicle, Ped owner);
 ```
 
-```
-how does this work?  
-```
+Disables or enables a specific vehicle weapon.
+This native needs to be called by every player, not just the driver. The weapon is only disabled for players who call the native, otherwise, if the driver shoots the weapon, other players can see it (but the driver himself can't).
 
-## Parameters
-* **disabled**: 
-* **weaponHash**: 
-* **vehicle**: 
-* **owner**: 
+An easier way to fully disable a vehicle's weapons would be removing the weapon from the vehicle's `handling.meta`.
 
+## Parameters
+* **disabled**: Whether the weapon should be disabled or not
+* **weaponHash**: The hash key of the weapon (vehicle weapons can be found here: https://wiki.rage.mp/index.php?title=Weapons#Vehicle_weapon_names)
```

Comment:
As suggested by Smallo, please refer only to the FiveM docs for such references.